source/vibe/core/log.d

```diff
@@ -941,8 +941,12 @@ private void doLog(S, T...)(LogLevel level, string mod, string func, string file
     nothrow
 {
 	try {
-		static if(T.length != 0)
-			auto args_copy = args;
+		static if(T.length != 0) {
+			if (!getLoggers().any!(l => l.minLevel <= level))
+				return;
+
+			T args_copy = args;
```

Comment:
won't compile because if T.length == 0 args_copy is undefined

Reply:
But this is inside of `static if (T.length != 0) { ... }`, so it should still work.

Reply:
Ah, I see, my fork is stuck in old version where formattedWrite is not in static if block